archetype/src/main/resources/archetype-resources/README.md

```diff
@@ -7,7 +7,17 @@ With Storm installed, you must first generate an uberjar:
 mvn clean package
 ```
 
-before submitting the topology using the storm command:
+You need to have an instance of URLFrontier running. See https://github.com/crawler-commons/url-frontier/tree/master/service for instructions; the easiest way is to use Docker.
```

Comment:
I suggest to write:

```md
See [the URLFrontier README](https://github.com/crawler-commons/url-frontier/tree/master/service); the easiest way is to use Docker.
```

Reply:
Should we link to https://github.com/DigitalPebble/stormcrawler-docker as a resource for an example `docker-compose` to run it straight out of the archetype?

Reply:
The docker-compose setup is useful but more complex to run. I'd rather keep the README for the archetype as simple as possible. I meant to add the commands to start URLFRontier with Docker.
I''ll add the link anyway for reference